server/config/api.ts

```diff
@@ -1 +1,2 @@
 export const BASE_URL = 'https://localhost:7210';
```

Comment:
Load this from env, this is enviroment specific URL

Reply:
Moved both the BASE_URL and GOOGLE_REFRESH_TOKEN_URL to env